rpcs3/Crypto/unzip.cpp

```diff
@@ -57,7 +57,7 @@ std::vector<u8> unzip(const void* src, usz size)
 
 	out.resize(zs.next_out - out.data());
 
-	res = inflateEnd(&zs);
+	inflateEnd(&zs);
```

Comment:
I'd like to keep this for potential debugging

Reply:
It's optimized away though